entrypoint.sh

```diff
@@ -24,7 +24,10 @@ else
     flask db upgrade -d $BASEDIR/migrations_test_db #this line used only if cloud test db is being used
     #flask db upgrade -d $BASEDIR/migrations_azure #this line used only if production database is being used 
 fi
-	
+
+
+# debugging only, to pass the CI build, start a dummy SMTP server that logs all emails to STDOUT
+python -m smtpd -c DebuggingServer -n localhost:1025 &
```

Comment:
I added a debugging SMTP server just to pass the CI. We need to remove it once we switch SMTP credentials over to the climatemind.org domain.